src/rgw/rgw_rest_s3.cc

```diff
@@ -951,6 +940,7 @@ void RGWListBucket_ObjStore_S3::send_versioned_response()
   s->formatter->close_section();
   rgw_flush_formatter_and_reset(s, s->formatter);
 }
+  s->formatter->close_section();
```

Comment:
i think this is crashing because it comes after the call to `rgw_flush_formatter_and_reset()`

Reply:
Makes sense. But the multisite tests are still failing

Reply:
@cbodley Works fine now. Could you please check